doc/user-guide.md

```diff
@@ -464,6 +465,47 @@ func main() {
 }
 ```
 
+##### If 3rd party resource does not have `AddToScheme()` function
+
+Use `AddKnownTypes()` and `AddToGroupVersion()` functions.
```

Comment:
@ytsarev If you could also please update this part and add a line about constructing a SchemeBuilder using the controller-runtime's utility.

Reply:
@hasbro17 sure, PR amended, thanks a lot for the catch.